poller/poller.go

```diff
@@ -8,17 +8,17 @@ import (
 
 	txTypes "github.com/cosmos/cosmos-sdk/types/tx"
 	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/likecoin/likechain/app"
 	"github.com/likecoin/likecoin-chain-tx-indexer/db"
 	"github.com/likecoin/likecoin-chain-tx-indexer/logger"
-	"github.com/likecoin/likecoin-chain/v2/app"
 	"github.com/tendermint/go-amino"
 	"github.com/tendermint/tendermint/libs/bytes"
 
 	tmTypes "github.com/tendermint/tendermint/types"
 )
 
 const batchSize = 1000
-const batchMaxHeightDiff = 100000
+const batchMaxHeightDiff = 100
```

Comment:
Why reduce this number?

Reply:
because the original value would take too long to flush the result into db. Once an error occurs, all the txs has polled have to be polled again. I think it is related to [#6](https://github.com/likecoin/likecoin-chain-tx-indexer/issues/6#issue-1027748172)

Reply:
updated, and also support env